pkg/schema/rtinstance/rtinstance.go

```diff
@@ -0,0 +1,7 @@
+package rtinstance
```

Comment:
Let's put this whole package under an `internal` dir. (so, `pkg/schema/internal/rt`.) We need to be doing a lot more of this in general, and this feels an especially good place to start.